features/bevragen/persoon/verblijfplaats/adres/verblijfadres/in-onderzoek.feature

```diff
@@ -0,0 +1,143 @@
+    | 081160                  |                        |               |           |                     |           | true        |              |               |         |            |            |              |                               |
+    | 081170                  |                        |               |           |                     |           |             |              | true          |         |            |            |              |                               |
+
+  Abstract Scenario: '<type>' is in onderzoek, maar veld '<veld naam>' wordt niet gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                 | waarde |
+    | gemeentecode (92.10) | 0519   |
+    | straatnaam (11.10)   | Spui   |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | <gevraagde fields>              |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam | waarde |
+    | type | Adres  |
+    En heeft de 'verblijfplaats' geen 'verblijfadres' gegevens
+
+    Voorbeelden:
+    | aanduiding in onderzoek | type                      | veld naam               | gevraagde fields                                     |
+    | 081115                  | naam openbare ruimte      | straat                  | verblijfplaats.verblijfadres.postcode                |
+    | 081120                  | huisnummer                | huisnummer              | verblijfplaats.verblijfadres.huisnummertoevoeging    |
+    | 081130                  | huisletter                | huisletter              | verblijfplaats.verblijfadres.huisnummer              |
+    | 081140                  | huisnummertoevoeging      | huisnummertoevoeging    | verblijfplaats.verblijfadres.straat                  |
+    | 081150                  | aanduiding bij huisnummer | aanduidingBijHuisnummer | verblijfplaats.verblijfadres.huisletter              |
+    | 081160                  | postcode                  | postcode                | verblijfplaats.verblijfadres.straat                  |
+    | 081170                  | woonplaatsnaam            | woonplaats              | verblijfplaats.verblijfadres.aanduidingBijHuisnummer |
+
+  Scenario: 'straatnaam' is in onderzoek, maar veld 'korteNaam' wordt niet gevraagd
```

Comment:
Deze naam komt bij mij wat vreemd over. Waarom niet:

` 'straatnaam' is in onderzoek, maar het veld 'straat' evenals het veld 'korteNaam' wordt niet gevraagd`

Reply:
> Deze naam komt bij mij wat vreemd over. Waarom niet:
> 
> ` 'straatnaam' is in onderzoek, maar het veld 'straat' evenals het veld 'korteNaam' wordt niet gevraagd`

Ik denk dat je hiermee zegt dat omdat straatnaam in onderzoek is, dat naam openbare ruimte in onderzoek is? Zo ja, is dit correct?

Reply:
Aanduiding in onderzoek is 081110. Dat betekent dat zowel straatnaam als korteNaam in onderzoek zijn dus de titel moet nog anders:

`'straatnaam' en 'korteNaam' zijn in onderzoek, maar de velden 'straat' en 'korteNaam' worden niet gevraagd`